docs/docs/unreachable.md

```diff
@@ -0,0 +1,143 @@
+the other as a major improvement ([MESOS-5948](https://issues.apache.org/jira/browse/MESOS-5948)). There is currently discussion in process to remove this rate limit default from DC/OS.
+
+## Marathon Unreachable Strategy
+
+Marathon has configuration options for working with unreachable tasks by setting the unreachable strategy for a Marathon application as part as the app definition:
+
+```json
+unreachableStrategy: {
+  "inactiveAfterSeconds": 0
+}
+```
+
+In order for this to take effect it is necessary to get a `TASK_UNREACHABLE` status update from Apache Mesos. The above constraints set the minimum time of managing an unreachable strategy event. The configuration for the app definition sets up to time events. The first is `inactiveAfterSeconds` which is the time in seconds after the `TASK_UNREACHABLE` event from Mesos that Marathon will create a replacement task. The second configuration is `expungeAfterSeconds` which is the amount of time after the `TASK_UNREACHABLE` that a task will be removed killed provided:
+
+1. it was replaced with the `inactiveAfterSeconds` event
+2. The original task (which was unreachable) becomes reachable again.
+
+From the time of `inactiveAfterSeconds` until the "expunge" event, the app will report in Marathon 2 of 1 for the number of tasks running for that app. "expunge" is a little odd, in that it is the time from `TASK_UNREACHABLE`. If the `inactiveAfterSeconds` trigger event trips, then the expunge will happen in the time configured regardless of if the task becomes reachable or not. In the case of a large delay in the
+return of the lost task, it will be killed immediately after it is seen by the system (assuming it is past expunge time).
+
+When the unreachable strategy is `{0, 0}`, the replacement is nearly immediate, with a replacement commonly occurring (for the test app) within roughly 3 seconds, and the expunge happening within 8-11 seconds. When we include this with the Mesos details described above for the loss of 1 agent, it means that Marathon will replace a task as soon as it is notified that the task is unreachable (which happens when the Mesos Master marks an agent as inactive), and it will expunge the original task as soon as it is reachable. If there is time configured for the strategy, then the event cycle that Marathon uses to managing if a task should be replaced or expunged is the task reconciliation cycle. The importance of this is 2 fold: 1) the unreachable strategy time has to expire and 2) the next task reconciliation needs to happen. The task reconciliation is a Marathon system configuration [`--reconciliation_interval`](https://mesosphere.github.io/marathon/docs/command-line-flags.html). The Marathon defined default is 10 minutes. This means that an unreachable strategy which includes `inactiveAfterSeconds` = 60, will have a task replaced between 60 seconds and 11 minutes. The reconciliation interval has the same effect on the expunging of a task.
```

Comment:
`1) the unreachable strategy... 2) the next...` - please use bullet points here

Reply:
Which unreachable strategy time. There are two.